### PR TITLE
Navigation: Add Import dashboard to quick actions

### LINKED ISSUE #65523: Add `Import dashboar` to quick add

### PR DESCRIPTION
**Pull request by ashharrison90**
**What is this feature?**

- adds `Import dashboard` to the quick add menu
- this also exposes it in the command palette as an action 🥳 

**Why do we need this feature?**

- better usability

**Who is this feature for?**

- anyone who wants to import a dashboard

**Which issue(s) does this PR fix?**:

Fixes https://github.com/grafana/grafana/issues/65523

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.
- [ ] There are no known compatibility issues with older supported versions of Grafana, or plugins.
- [ ] It passes the [Hosted Grafana feature readiness review](https://docs.google.com/document/d/1QL9Ly8KnXzpb6ISbg49pTODRO5mhA5tkkfIZVX6pqQU/edit) for observability, scalability, performance, and security.

### CODE REVIEW

File: pkg/services/navtree/navtreeimpl/navtree.go [hunk 1]
```diff
@@ -402,6 +402,13 @@ func (s *ServiceImpl) buildDashboardNavLinks(c *contextmodel.ReqContext, hasEdit
 				Text: "New dashboard", Icon: "plus", Url: s.cfg.AppSubURL + "/dashboard/new", HideFromTabs: true, Id: "dashboards/new", ShowIconInNavbar: true, IsCreateAction: true,
 			})
 		}
+
+		if hasAccess(hasEditPermInAnyFolder, ac.EvalPermission(dashboards.ActionDashboardsCreate)) {
```
Comment (JoaoSilvaGrafana) [inline pkg/services/navtree/navtreeimpl/navtree.go:406]: This is the exact same if as above no? Can we just merge them?

File: public/locales/fr-FR/grafana.json [hunk 2]
```diff
@@ -38,7 +38,7 @@
       "query-tab": "Requête",
       "stats-tab": "Statistiques",
       "subtitle": "{{queryCount}} requêtes avec un délai total de requête de {{formatted}}",
-      "title": "Inspecter : {{panelTitle}}"
+      "title": "Inspecter\u00a0: {{panelTitle}}"
```
Comment (JoaoSilvaGrafana) [inline public/locales/fr-FR/grafana.json:41]: What happened to all these french translations?

**Review by JoaoSilvaGrafana — APPROVED** [review]
Looks great apart from two nits! 👏

File: pkg/services/navtree/navtreeimpl/navtree.go (hunk 1 above)
Comment (ashharrison90) [reply pkg/services/navtree/navtreeimpl/navtree.go:406]: oh yeah, idk how i missed that! 😅

File: public/locales/fr-FR/grafana.json (hunk 2 above)
Comment (ashharrison90) [reply public/locales/fr-FR/grafana.json:41]: that's the output when running `yarn i18n:extract`... i asked @joshhunt and he said it was correct to commit it, crowdin will "fix" it later

**Review by JoaoSilvaGrafana — APPROVED** [review]
👏

**Review by joshhunt — APPROVED** [review]

**Pull request merged**